src/scilpy/cli/scil_volume_resample.py

```diff
@@ -102,9 +102,19 @@ def main():
     ref_img = None
     if args.ref:
         ref_img = nib.load(args.ref)
+        
         # Must not verify that headers are compatible. But can verify that, at
-        # least, the last columns of their affines are compatible.
-        if not np.array_equal(img.affine[:, -1], ref_img.affine[:, -1]):
+        # least, the first columns of their affines are compatible.
+        img_zoom_invert = []
+        ref_zoom_invert = []
+        for index in range(len(img.header.get_zooms())):
+            img_zoom_invert.append(1/img.header.get_zooms()[index])
```

Comment:
I would prefer: `img_zoom_invert = [1 / zoom for zoom in img.header.get_zooms()]` (and same for the next)